docs/index.md

```diff
@@ -29,10 +33,8 @@ compute backends.
    :members:
 ```
 
-:::{note}
-REANA-Job-Controller supports the Kubernetes job manager by default, no need
-to pass any build argument.
-:::
+:::{note} REANA-Job-Controller supports the Kubernetes job manager by default,
```

Comment:
This could break the display on the ReadTheDocs web site. I'll remove the special note formatting whilst merging, for the sake of simplicity.